Costura.dll does not have a strong name

When compiling a strong named assembly that uses Costura, the following warning appears
```
CSC : warning CS8002: Referenced assembly 'Costura, Version=5.7.0.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.
```

## Please check all of the platforms you are having the issue on (if platform is not listed, it is not supported)
 - [ ] WPF
 - [ ] UWP
 - [ ] iOS
 - [ ] Android
 - [x] .NET Standard
 - [x] .NET Core

## Component
NA

## Version of Library
5.7.0.0

## Version of OS(s) listed above with issue
NA

## Steps to Reproduce
Build an assembly with a strong name

## Expected Behavior
No warnings

## Actual Behavior
```
CSC : warning CS8002: Referenced assembly 'Costura, Version=5.7.0.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.
```

--- a/src/Costura/Costura.csproj
+++ b/src/Costura/Costura.csproj
@@ -10,6 +10,7 @@
     <PackageId>Costura.Fody</PackageId>
     <PackageTags>ILMerge, ILWeaving, Embed, Resource, Fody, Cecil</PackageTags>
     <DisableFody>true</DisableFody>
+    <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)key.snk</AssemblyOriginatorKeyFile>
     <NoWarn>$(NoWarn);NU5100</NoWarn>
     <NoError>$(NoError);NU5100</NoError>

